specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2020-01-01/SecurityInsights.json

```diff
@@ -664,364 +939,1104 @@
+      "properties": {
+        "logs": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/DataConnectorDataTypeCommon"
+            }
+          ],
+          "description": "Logs data type.",
+          "type": "object"
         }
-      ],
-      "description": "ASC (Azure Security Center) data connector properties.",
+      },
+      "type": "object"
+    },
+    "AwsCloudTrailDataConnectorProperties": {
+      "description": "Amazon Web Services CloudTrail data connector properties.",
       "properties": {
-        "subscriptionId": {
-          "description": "The subscription id to connect to, and get the data from.",
+        "awsRoleArn": {
+          "description": "The Aws Role Arn (with CloudTrailReadOnly policy) that is used to access the Aws account.",
           "type": "string"
+        },
+        "dataTypes": {
+          "$ref": "#/definitions/AwsCloudTrailDataConnectorDataTypes",
+          "description": "The available data types for the connector."
         }
       },
       "type": "object"
     },
-    "ActionRequest": {
+    "Bookmark": {
       "allOf": [
         {
           "$ref": "#/definitions/ResourceWithEtag"
         }
       ],
-      "description": "Action for alert rule.",
+      "description": "Represents a bookmark in Azure Security Insights.",
       "properties": {
         "properties": {
-          "$ref": "#/definitions/ActionRequestProperties",
-          "description": "Action properties for put request",
+          "$ref": "#/definitions/BookmarkProperties",
+          "description": "Bookmark properties",
           "x-ms-client-flatten": true
         }
       },
       "type": "object"
     },
-    "ActionPropertiesBase": {
-      "description": "Action property bag base.",
+    "BookmarkList": {
+      "description": "List all the bookmarks.",
       "properties": {
-        "logicAppResourceId": {
-          "description": "Logic App Resource Id, providers/Microsoft.Logic/workflows/{WorkflowID}.",
+        "nextLink": {
+          "description": "URL to fetch the next set of cases.",
+          "readOnly": true,
           "type": "string"
+        },
+        "value": {
+          "description": "Array of bookmarks.",
+          "items": {
+            "$ref": "#/definitions/Bookmark"
+          },
+          "type": "array"
         }
       },
       "required": [
-        "logicAppResourceId"
-      ],
-      "type": "object"
+        "value"
+      ]
     },
-    "ActionRequestProperties": {
-      "allOf": [
-        {
-          "$ref": "#/definitions/ActionPropertiesBase"
-        }
-      ],
-      "description": "Action property bag.",
+    "BookmarkProperties": {
+      "description": "Describes bookmark properties",
       "properties": {
-        "triggerUri": {
-          "description": "Logic App Callback URL for this specific workflow.",
+        "created": {
```

Comment:
is this readonly property? I am assuming this is generated by the service. ANd the user cant set this. If so, please mark readOnly:true

Reply:
How was this comment resolved?

Reply:
What is the problem here?

Reply:
@markcowl , Can you please point the problem here?

Reply:
This comment from the arm reviewer, that the property should be marked as read-only has not been addressed.  I assume that you resolved it some other way, and want to know what that is.

Reply:
Hi,
the triggerUri is not a readonly property. the user supplied this property and the user can change it as well